test/engine_test.dart

```diff
@@ -188,7 +188,7 @@ void defineLinterEngineTests() {
   });
 }
 
-typedef NodeVisitor(node);
+typedef NodeVisitor = void Function(AstNode node);
```

Comment:
Regarding its usage, it can be call with `PSEntry` at https://github.com/dart-lang/linter/blob/33a9a0a9df49497bd9bca20c0cb2688e5937db05/test/engine_test.dart#L231

Reply:
Ah!  Thanks.

Reply:
(Obviously a bit hasty.)